src/Tizen.Applications.Common/Tizen.Applications.Common.csproj

```diff
@@ -4,6 +4,10 @@
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(NOTIZEN)'=='true'">
+    <DefineConstants>$(DefineConstants);NOTIZEN</DefineConstants>
+  </PropertyGroup>
+
```

Comment:
These lines can be moved to `build/common.props`
 the `build/common.props` is processed before each module's csproj.

Reply:
Much better, thanks a lot for the review.